### PR TITLE
.set width int argument command

### PR DESCRIPTION
**Pull request by Arman199116**

### CODE REVIEW

File: postgresql-11.5/src/bin/psql/commandbteq.c [hunk 1]
```diff
@@ -214,6 +219,8 @@ exec_command(const char *cmd,
         status = exec_command_set(scan_state, active_branch);
     } else if (strcasecmp(cmd, "quit") == 0 || strcasecmp(cmd, "q") == 0) {
         status = exec_command_quit(scan_state, active_branch);
+    } else if (strcmp(cmd, "c") == 0 || strcmp(cmd, "connect") == 0) {
+        status = exec_command_connect(scan_state, active_branch);
```
Comment (maryantonyan) [inline postgresql-11.5/src/bin/psql/commandbteq.c:223]: Why do we need this?

File: postgresql-11.5/src/bin/psql/commandbteq.c [hunk 2]
```diff
@@ -796,4 +788,361 @@ process_file_bteq(char *filename, bool use_relative_path)
...
+ *
+ * .c - - hst        Connect to current database on current port of
+ *                    host "hst" as current user.
+ * .c - usr - prt    Connect to current database on port "prt" of current host
+ *                    as user "usr".
+ * .c dbs            Connect to database "dbs" on current port of current host
+ *                    as current user.
+ */
+static dotResult
+exec_command_connect(BteqScanState scan_state, bool active_branch)
```
Comment (maryantonyan) [inline postgresql-11.5/src/bin/psql/commandbteq.c:809]: Why is this functionality added in scope of current PR?
As far as I know, .SET WIDTH command is not related to the database connection.

File: postgresql-11.5/src/bin/psql/settings.h [hunk 3]
```diff
@@ -90,6 +90,7 @@ typedef struct _psqlSettings
 	PGresult   *last_error_result;	/* most recent error result, if any */
 
 	printQueryOpt popt;
+    printQueryOptBteq popt_bteq;
```
Comment (maryantonyan) [inline postgresql-11.5/src/bin/psql/settings.h:93]: correct alignment

File: postgresql-11.5/src/bin/psql/startup.c [hunk 4]
```diff
@@ -171,9 +171,23 @@ main(int argc, char *argv[])
 
     refresh_utf8format(&(pset.popt.topt));
 
+    pset.popt_bteq.topt.format = PRINT_ALIGNED;
+    pset.popt_bteq.topt.border = 1;
+    pset.popt_bteq.topt.pager = 1;
+    pset.popt_bteq.topt.pager_min_lines = 0;
+    pset.popt_bteq.topt.start_table = true;
+    pset.popt_bteq.topt.stop_table = true;
+    pset.popt_bteq.topt.default_footer = true;
```
Comment (maryantonyan) [inline postgresql-11.5/src/bin/psql/startup.c:180]: do we need all this variables?

**Review by maryantonyan — COMMENTED** [review]
I have a segfault issue on this branch. Please fix it, and then I will be able test your changes before approving PR.
On master branch:
image

On current branch:
image

File: postgresql-11.5/src/bin/psql/startup.c (hunk 4 above)
Comment (Arman199116) [reply postgresql-11.5/src/bin/psql/startup.c:180]: yes, because we need to use a pset printQueryOptBteq my_popt_bteq

File: postgresql-11.5/src/bin/psql/commandbteq.c (hunk 2 above)
Comment (Arman199116) [reply postgresql-11.5/src/bin/psql/commandbteq.c:809]: it was added to allow us to join database to use tables

File: postgresql-11.5/src/bin/psql/commandbteq.c (hunk 1 above)
Comment (Arman199116) [reply postgresql-11.5/src/bin/psql/commandbteq.c:223]: it was added to allow us to join database to use tables

Comment (maryantonyan) [reply postgresql-11.5/src/bin/psql/commandbteq.c:223]: After executing LOGON command you should already be able to use the database to which you are connected and all its tables.

The LOGON command is supposed to connect to the database and nothing else.
